client/modules/workspace/src/AppsSubmissionDetails/AppsSubmissionDetails.tsx

```diff
@@ -259,7 +259,7 @@ export const AppsSubmissionDetails: React.FC<{
             htmlType="submit"
             disabled={!isValid}
             loading={isSubmitting}
-            style={{ width: 130 }}
+            style={{ width: 150 }}
```

Comment:
Note: increased to 150 to accommodate loading spinner with longer "Launch Session" text